File: questionOne.c
/*
 *Project Euler multiples of 3 and 5
 *Author: Matthew Zheng
 */

#include <stdlib.h>
#include <stdio.h>

int main(void){
	int mySum = 0;

	for (int i = 0; i < 1000; i++){
		if (i % 5 == 0 || i % 3 == 0){
			mySum += i;
			printf("%d\n", i);
		}
	}

	printf("The sum is %d\n", mySum);

	system("PAUSE");
	return 0;

}

File: questionOne.py
#!/usr/bin/env python
_author_ = "Matthew Zheng"
_purpose_ = "Euler question 1, multiples of 3 and 5"

import sys

def main():
    mySum = 0
    for num in range (0, 1000):
        if num % 5 == 0 or num % 3 == 0:
            mySum += num
            print(num)

    #print final result
    print("The sum is", mySum)

if __name__ == "__main__":
    main()
